apisix/init.lua

```diff
@@ -698,8 +698,8 @@ local function healcheck_passive(api_ctx)
         end
     end
 
-    local http_statuses = passive and passive.unhealthy and
-                          passive.unhealthy.http_statuses
+    http_statuses = passive and passive.unhealthy and
+            passive.unhealthy.http_statuses
```

Comment:
Err. It would be better if the `passive` can be aligned with the `= passive` in the previous line, just like the state before your change.

Reply:
ok. Got it.